### ISSUE TITLE
Add a new field (IsActive,bit default 1) to the Yaf_Category table

### ISSUE THREAD

**Issue by daprose-dotcom** [YAFNET/YAFNET#694]
PLease Add a new field (IsActive, Type bit, Default Value=1) to the Yaf_Category table

Since the default value - 1, nothing will change in every installation, all categories will have teh IsActive = 1

However, Admin can set that field IsActive = 0, in such a case, it will not show on the board but it will exist in the database anyway. No forums should be displayed for categories with the IsActive field set to 0

**Comment by w8tcha** [comment]
There exist the forum setting "hide if no access" then the category would not show up. If there is no forum. Would that not solve the same purpose?

**Comment by daprose-dotcom** [comment]
No. That covers other type of issue. My proposal is for a temporary hold/disable of categories that I do have forums, topics and replies, but for some reason, I (temporarily) do not want to display that category on the board. I could be more than one category that I may not want to display!  

This feature is necessary for boards that share the exact same database, but one board may not want to display certain categories as the other does.

**Comment by w8tcha** [comment]
I given up arguing with you. I can only tell you how yaf works. In yaf each board has its own categories and forums.

If you create a second, third board then it will not have the same categories as the first board. So your column will not work!

**Comment by daprose-dotcom** [comment]
It should work on as many boards as I add because they all use the same database structure.

**Comment by w8tcha** [comment]
But then you need to create your own customized version, or find another forum softwae. 

You have to understand that I can't rewrite the entire software to fit your personal needs.

Boards inside the db don't share any data, and this won't change

**Comment by daprose-dotcom** [comment]
Even if we do not share any data, we still need that field to enable or disable (temporarily) one or more specific categories on our board. I do not want to have to DELETE a category and recreate it again when I do not want it to display for some reason. For Ex: A court order, a litigation, a police investigation, aggravated posts, out of order attacks, malicious posts, etc.

**Comment by daprose-dotcom** [comment]
Please check:

http://daprose.com/forums
http://aspnetportl.xyz

They both are sharing the same database ..

**Comment by daprose-dotcom** [comment]
> You have to understand that I can't rewrite the entire software to fit your personal needs.

We are NOT rewriting the entire software, we are just making it better, little changes!! in fact only one or two lines changing it one at a time. Your forums will be the best ever in the whole planet!!

**Comment by w8tcha** [comment]
> Please check:
> 
> 
> 
> http://daprose.com/forums
> 
> http://aspnetportl.xyz
> 
> 
> 
> They both are sharing the same database ..
> 
> 
> 
> 

The second url doesn't work. Is it possible that we talk about to different issue? If both use the same db then under host > boards there should be to boards listest correct?

**Comment by daprose-dotcom** [comment]
It is working now. Sorry, I was making a change in one of them. Just refresh the page or close your browser, remove history and delete cache. Thank you

**Comment by w8tcha** [comment]
> It is working now. Sorry, I was making a change in one of them. Just refresh the page or close your browser, remove history and delete cache. Thank you
> 
> 

If both use the same db then under host > boards there should be to boards listest correct?

**Comment by daprose-dotcom** [comment]
> The second url doesn't work. Is it possible that we talk about to different issue? If both use the same db then under host > boards there should be to boards listed correct?

They, I don't want to break you down with so many requests but yes, some boards will be local, we do not mess with them, but they will share the same DB structure, that is all.  other board will be DEFAULT BOARD #1 This board will be controlled by YAFNET, this is the INTERNATIONAL board.

Meaning that, IN THE FUTURE, (When we finish will all additions), you will not only provide the Forums ( the YafNet Application), but also the WORLD DATA CONNECTION.  Being said that, you will destroy FACEBOOK, USENET, TWITTER, etc...

**Comment by daprose-dotcom** [comment]
Sorry, made a typo mistake: http://aspnetportal.xyz

**Comment by daprose-dotcom** [comment]
After making the last change for the statistics, now the right column disappeared. 
it is enabled in the Host Settings -> Display  but it doesn't show anymore

**Comment by daprose-dotcom** [comment]
The Cookies must be UNIQUE for each APP setup, I think the reason the statistics do not show up anymore is because some cookies have the same names in all installations. To overcome that problem we need to add a SALT either in the APP.CONFIG or in the GLOBAL.ASAX  for each installation.

**Comment by daprose-dotcom** [comment]
> If both use the same db then under host > boards there should be to boards listest correct?

Yes, yes, yes, the boards will list correctly because all new installations will have a shared database, however, the same way we added the IsActive field in the Yaf_Categories, we should add a IsLocal (bit = 1) in the boards table. this changes WILL NOT AFFECT ANY PREVIOUS INSTALLATIONS because in the database structure the default VALUE in both cases is set to 1
